### PR TITLE
Answers yes reported

### PR DESCRIPTION
**Pull request by kssampson**
- added functionality and css to Load More Questions

### CODE REVIEW

File: client/src/containers/QuestionsAnswers.jsx [hunk 1]
```diff
@@ -9,6 +9,9 @@ const QuestionsAnswers = ( {itemId} ) => {
   const [questionData, setQuestionData] = useState([]);
   const [resultsToShow, setResultsToShow] = useState([])
   const [currentCount, setCurrentCount] = useState(2);
+  const [numOfQuestionToGet] = useState(400);
+
+  console.log('resultsToShow.length', resultsToShow.length)
```
Comment (LukeLarson2) [inline client/src/containers/QuestionsAnswers.jsx:14]: please remove this console.log

File: client/src/containers/QuestionsAnswers.jsx [hunk 2]
```diff
@@ -40,20 +43,28 @@ const QuestionsAnswers = ( {itemId} ) => {
 
   useEffect(() => {
     setResultsToShow(questionData.slice(0, currentCount))
-  }, [questionData])
+    console.log('quesitonData', questionData)
```
Comment (LukeLarson2) [inline client/src/containers/QuestionsAnswers.jsx:46]: please remove this console.log

File: client/src/containers/QuestionsAnswers.jsx [hunk 3]
```diff
@@ -40,20 +43,28 @@ const QuestionsAnswers = ( {itemId} ) => {
...
+  }, [questionData, currentCount])
 
 
-  const handleClick = () => {
+  const handleLoadMoreQuestions = () => {
+    console.log('in load more answers::::', currentCount)
```
Comment (LukeLarson2) [inline client/src/containers/QuestionsAnswers.jsx:51]: please remove this conosle.log and the one on the following line

**Review by LukeLarson2 — CHANGES_REQUESTED** [review]
please remove the commented console.logs and resubmit

File: client/src/components/questions_answers/Answers.jsx [hunk 4]
```diff
@@ -6,6 +6,22 @@ const Answers = ( { questionId } ) => {
 
   const [answerData, setAnswerData] = useState([]);
   const [marked, setMarked] = useState({});
+  const [reported, setReported] = useState({});
+
+  const checkReprted = (id) => {
```
Comment (clee4037) [inline client/src/components/questions_answers/Answers.jsx:11]: FYI, small typo, but spelling is consistent throughout file

File: client/src/containers/QuestionsAnswers.jsx (hunk 3 above)
Comment (kssampson) [reply client/src/containers/QuestionsAnswers.jsx]: updated

File: client/src/containers/QuestionsAnswers.jsx (hunk 2 above)
Comment (kssampson) [reply client/src/containers/QuestionsAnswers.jsx]: updated

File: client/src/components/questions_answers/Answers.jsx [hunk 5]
```diff
@@ -73,10 +88,10 @@ const Answers = ( { questionId } ) => {
             <div className="k-answer-date">{formatDate(answer.date)} | </div>
 
             <div className="k-answer-helpful">
-              Helpful? <span className="k-answer-yes-click" onClick={() => checkMarked(id)}>Yes</span> |  ({answer.helpfulness})
+              Helpful? <span className="k-answer-yes-click" style={{cursor: marked[id] ? "default" : "pointer"}} onClick={() => checkMarked(id)}>Yes</span> |  ({answer.helpfulness})
             </div>
             <div className="k-answer-report">
-              <span className="k-answer-report-click">Report</span>
+              <span className="k-answer-report-click" onClick={() => checkReprted(id)} style={{cursor: reported[id] ? "defualt" : "pointer"}} >{reported[id] ? "Reported" : "Report"}</span>
```
Comment (clee4037) [inline client/src/components/questions_answers/Answers.jsx:94]: typo: "checkReprted"

**Review by clee4037 — COMMENTED** [review]
Few comments. Other than that, looks good!

File: client/src/components/questions_answers/Answers.jsx (hunk 5 above)
Comment (kssampson) [reply client/src/components/questions_answers/Answers.jsx]: Changed. Found another in the pointer on line 94. Also changed.

**Review by LukeLarson2 — APPROVED** [review]
good to go, thank you for making the corrections

**Pull request merged**